package.json

```diff
@@ -28,15 +28,15 @@
   "author": "amasad",
   "license": "MIT",
   "dependencies": {
+    "@cnakazawa/watch": "^1.0.3",
```

Comment:
this is good, but I checked the repo an it seems you didn't push the `package.json` update. Is it possible for your todo so and tag the release?

Reply:
Tagged as v1.0.3 on https://github.com/cpojer/watch/releases and pushed to a `new-master` branch. I wanted to keep the `master` branch without this change because I have an open PR against the real `watch` repo and will update sane back to it if it ever gets merged and published.